package.json

```diff
@@ -6,19 +6,22 @@
 	"scripts": {
 		"dev": "vite",
 		"build": "vite build",
-		"fmt": "biome format .",
-		"fmt:fix": "biome format --write .",
-		"lint:biome": "biome check",
+		"fix": "concurrently \"bun:fix:*\"",
+		"fix:biome:format": "biome format --write .",
+		"fix:biome:check": "biome check --write *",
```

Comment:
Running a bunch of processes that write to random files simultaneously seems ill-advised. Maybe do this in a linear order?

eslint && biome-lint && biome-format